m2c/evaluate.py

```diff
@@ -1155,6 +1197,8 @@ def handle_rlwinm(
 
         if simplify:
             lower_bits = replace_clz_shift(fold_divmod(lower_bits))
+            # This really only makes sense if the clz simplification didn't happen
```

Comment:
```suggestion
```

(this kind of "it only makes sense if so and so" is common, and the check for `>>` at the top of the function will take care of it)